pkg/handler/enqueue_static_handler.go

```diff
@@ -27,8 +29,11 @@ type BroadcastRequests struct {
 	WorkQueues *skqueue.MultiClusterQueues
 }
 
+// TODO: (sam-heilbron)
```

Comment:
Let's remove this as we agreed it's not relevant

Reply:
done